modules/ROOT/pages/spring/spring-module.adoc

```diff
@@ -5,6 +5,20 @@ endif::[]
 
 The Spring module allows Mule apps to make use of the Spring Framework.
 
+== To Install the Spring module
+
+Add it to your application using the Studio palette or add the following dependency in your `pom.xml` file:
```

Comment:
Please be specific. Instead of using "it", use "To add the Spring Module to your application, use ..." 
Also, add directions about how to install the Spring Module from Exchange:

. In Anypoint Studio, click the Exchange icon in the Studio task bar.
. Click Login in Anypoint Exchange and supply your Anypoint Platform username and password.
. Search for Spring Module and click Install.
. Follow the prompts to install the connector.

Reply:
Fixed in d1b2c1e